StonksWeb/SmartSaver.aspx.cs

```diff
@@ -14,6 +14,14 @@ public partial class SmartSaver : Page
 
         protected void Page_Load(object sender, EventArgs e)
         {
+            HousingSlider.Value = FinancialPlanController.ActivePlan.GetExpense(ExpenseType.Housing).Value.ToString();
```

Comment:
Would be better to map sliders to corresponding expense types, like done for textboxes. Also need null check for expenses (they can sometimes be undefined)